app/components/filmstrip.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { DISCIPLINES } from './disciplines';
+
+export default class Filmstrip extends React.Component {
+  constructor(props) {
```

Comment:
Just a FYI, passing `props` in the constructor is not necessary https://stackoverflow.com/questions/30571875/whats-the-difference-between-super-and-superprops-in-react-when-using-e